R/model.R

```diff
@@ -1,6 +1,6 @@
 #' Create a new CmdStanModel object
 #'
-#' @description \if{html}{\figure{logo.png}{options: width=25}}
+#' @description \if{html}{\figure{logo.svg}{options: width=25}}
```

Comment:
I think this is the only necessary change in the model.R file but there's a huge diff for the file due to a bunch of linting. Can you remove that from this PR and just keep the logo change in model.R?